stylesheets/_font_stack.scss

```diff
@@ -10,10 +10,12 @@
 //  the font-stack in a font-face declaration rather
 //  than with the usual font-family.
 
+// scss-lint:disable NameFormat
```

Comment:
It'd be good to include a reason with each disabled linter. I'd like to enable this linter in the future: https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#disablelinterreason